src/core_seaice/shared/mpas_seaice_forcing.F

```diff
@@ -1622,29 +1626,41 @@ end subroutine post_oceanic_coupling
 !>
 !-----------------------------------------------------------------------
 
-  subroutine init_data_iceberg_forcing(domain)
+  subroutine init_data_iceberg_forcing(domain, clock)
 
     type (domain_type) :: domain
 
+    type (MPAS_Clock_type) :: clock
+
     logical, pointer :: &
          config_do_restart
 
     character(len=strKIND) :: &
          forcingIntervalMonthly, &
          forcingReferenceTimeMonthly
 
+!     type (MPAS_clock_type) :: simulationClock
+     type (MPAS_Time_Type) :: currTime
+     character(len=strKIND) :: timeStamp
+     integer :: ierr
+
     ! get configuration options
     call MPAS_pool_get_config(domain % configs, "config_do_restart", config_do_restart)
 
     forcingIntervalMonthly = "00-01-00_00:00:00"
     forcingReferenceTimeMonthly = "0001-01-15_00:00:00"
 
+!    currTime = mpas_get_clock_time(simulationClock, MPAS_NOW, ierr)
+    currTime = mpas_get_clock_time(clock, MPAS_NOW, ierr)
+    call mpas_get_time(curr_time=currTime, dateTimeString=timeStamp, ierr=ierr)
+    timeStamp = '0000'//trim(timeStamp(5:))
+
     ! create own data iceberg forcing group
     call MPAS_forcing_init_group(&
          seaiceForcingGroups, &
          "seaice_data_iceberg_forcing_monthly", &
          domain, &
-         '0000-01-01_00:00:00', &
+         timeStamp, &
```

Comment:
This only effects an initial run not a restart run. I guess the code assumes the initial simulation (but not restarts) start on the year start. Is this the issue?

Reply:
@darincomeau: Can you email me the preprocessed files mpas_forcing.f90 and mpas_seaice_forcing.f90 so I can see what line numbers the traceback corresponds to?